### PR TITLE
Sentry progress deadline

### PR DESCRIPTION
**Pull request by thetechnick**

### CODE REVIEW

File: cmd/package-operator-manager/main.go [hunk 1]
```diff
@@ -53,12 +55,23 @@ func main() {
 	flag.StringVar(&opts.probeAddr, "health-probe-bind-address", ":8081",
 		"The address the probe endpoint binds to.")
 	flag.BoolVar(&opts.printVersion, "version", false, "print version information and exit.")
+	flag.StringVar(&opts.sentryDSN, "sentry-dsn", os.Getenv("PKO_SENTRY_DSN"), "Sentry DSN.")
```
Comment (Nanyte25) [inline cmd/package-operator-manager/main.go:58]: @thetechnick this is where the DSN and config gets applied?

File: config/static-deployment/deployment.yaml.tpl [hunk 2]
```diff
@@ -26,6 +26,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: PKO_SENTRY_DSN
+          value: https://2a9bea4b2ee04fafa940ee32b0e4660b@glitchtip.stage.devshift.net/17
```
Comment (Nanyte25) [inline config/static-deployment/deployment.yaml.tpl:30]: @thetechnick I am not sure that we should be applying the DNS directly like this ?

**Comment by Nanyte25** [comment]
@apahim and @thetechnick can you approve this PR

**Comment by Nanyte25** [comment]
@thetechnick was their some TLS issue with the image you where using?

File: apis/core/v1alpha1/clusterobjectdeployment_types.go [hunk 3]
```diff
@@ -9,6 +9,12 @@ type ClusterObjectDeploymentSpec struct {
 	// Number of old revisions in the form of archived ObjectSets to keep.
 	// +kubebuilder:default=10
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+	// The maximum time in seconds for a ObjectDeployment to make progress before it
+	// is considered to be failed. The ObjectDeployment controller will continue to
+	// process failed objectdeployments and a condition of type ProgressDeadlineExceeded
+	// will be surfaced in the ObjectDeployment status.
+	// +kubebuilder:default=600
+	ProgressDeadlineSeconds *int32 `json:"progressDeadlineSeconds,omitempty"`
```
Comment (garrettthomaskth) [inline apis/core/v1alpha1/clusterobjectdeployment_types.go:17]:
```suggestion
	// The maximum time in seconds for a ClusterObjectDeployment to make progress before it
	// is considered to be failed. The ClusterObjectDeployment controller will continue to
	// process failed ClusterObjectDeployments and a condition of type ProgressDeadlineExceeded
	// will be surfaced in the ClusterObjectDeployment status.
	// +kubebuilder:default=600
	ProgressDeadlineSeconds *int32 `json:"progressDeadlineSeconds,omitempty"`
```

**Pull request closed without merging**